content/en/docs/hertz/overview/_index.md

```diff
@@ -7,45 +7,50 @@ description: >
+
+Nowadays, as Go gain popularity in microservice development, Hertz will be the right choice if you are looking for a customizable, high-performance framework to support a variety of use case.
 
 ## Architecture
 ![HERTZ](/img/docs/hertz.png)
 
-## Basic Features
-- High usability
+## Features
+### High usability
 
-  During the development process, it is often more important to write the correct code quickly. Therefore, in the iterative process of Hertz, we actively listen to users' opinions and continue to polish the framework, hoping to provide users with a better user experience and help users write correct code faster.
+In modern software engineering, it is agreed that the delivery of high-quality code in a short period of time has become more important in a highly competitive environment. With that in mind, in the initial interactions of Hertz, we actively listen to users' feedback and endeavour to polish the framework to improve user experience and help developers to get the job done quickly and correctly.
 
 - High performance
+Hertz uses Netpoll, a high-performance network library built from sketch, by default. In comparison to go net implementation, the benchmark indicates that in some scenarios, Hertz's performance is better in terms of both QPS and time delay. 
 
-  Hertz uses the self-developed high-performance network library Netpoll by default. In some special scenarios, compared to Go Net, Hertz has certain advantages in QPS and time delay. For performance data, please refer to the Echo data in the figure below.
-  ![Performance](/img/docs/hertz_performance.png)
-  For detailed performance data, please refer to https://github.com/cloudwego/hertz-benchmark .
+The following diagrams show how Hertz performs in comparison with other popular frameworks for echo requests.
+ ![Performance](/img/docs/hertz_performance.png)
+Please refer to https://github.com/cloudwego/hertz-benchmark for more details about benchmarking.
 
-- High extensibility
+### High extensibility
 
-  Hertz adopts a layered design, providing more interfaces and default extension implementations. Users can also extend by themselves. At the same time, thanks to the layered design of the framework, the extensibility of the framework will be much greater. At present, only stable capabilities are open-sourced to the community. More planning refers to [RoadMap](https://github.com/cloudwego/hertz/blob/main/ROADMAP.md).
+Hertz adopts a layered architecture to deliver a "batteries included" experience. Due to its layered design, the framework is high extensible while its core functionality remains robust. Hertz comes with default implementations for many modules but also enables users to extend them to fit their own needs. At present, only stable capabilities have been made available to the open source community. More planning refers to [RoadMap](https://github.com/cloudwego/hertz/blob/main/ROADMAP.md).
```

Comment:
highly extensible ?

Reply:
@GuangmingLuo  Sorry for the delay. Would you like to take a look at changes I've made?